### PR TITLE
48855 style fix pylint issues pointless string statement

### PR DESCRIPTION
**Pull request by seanjedi**
- [ ] Accomplishes a task for #48855
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

File: pandas/tests/io/parser/common/test_read_errors.py [hunk 1]
```diff
@@ -242,7 +242,7 @@ def test_bad_header_uniform_error(all_parsers):
     msg = "Expected 2 fields in line 2, saw 4"
     if parser.engine == "c":
         msg = "Could not construct index. Requested to use 1 "
-        "number of columns, but 3 left to parse."
+        # number of columns, but 3 left to parse.
```
Comment (MarcoGorelli) [inline pandas/tests/io/parser/common/test_read_errors.py:245]: This needs joining with the string above it, it probably wasn't intended as a comment

Could you then also check that the test still passes please?

**Review by MarcoGorelli — COMMENTED** [review]
Thanks for working on this

File: pandas/tests/indexing/interval/test_interval_new.py [hunk 2]
```diff
@@ -163,7 +163,7 @@ def test_loc_with_overlap(self, indexer_sl):
         tm.assert_series_equal(expected, result)
 
         msg = "'can only get slices from an IntervalIndex if bounds are"
-        " non-overlapping and all monotonic increasing or decreasing'"
+        # non-overlapping and all monotonic increasing or decreasing'
```
Comment (MarcoGorelli) [inline pandas/tests/indexing/interval/test_interval_new.py:166]: likewise here

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]

File: pandas/tests/indexing/interval/test_interval_new.py (hunk 2 above)
Comment (seanjedi) [reply pandas/tests/indexing/interval/test_interval_new.py]: Sure thing! I made the change to make it a string again, and retested the code
image

I have made the change as part of: efe54026d6608b13160315b5f6560f3174657302

File: pandas/tests/io/parser/common/test_read_errors.py (hunk 1 above)
Comment (seanjedi) [reply pandas/tests/io/parser/common/test_read_errors.py]: Sure thing! I made the change to make it a string again, and retested the code
image

I have made the change as part of: efe54026d6608b13160315b5f6560f3174657302

**Review by MarcoGorelli — CHANGES_REQUESTED** [review]
thanks for updating! just got a few more comments

File: pandas/tests/frame/methods/test_reset_index.py [hunk 3]
```diff
@@ -453,7 +453,7 @@ def test_reset_index_duplicate_columns_allow(
         else:
             if not flag and allow_duplicates:
                 msg = "Cannot specify 'allow_duplicates=True' when "
-                "'self.flags.allows_duplicate_labels' is False"
+                # 'self.flags.allows_duplicate_labels' is False
```
[the remark this answers is not in the document]
Comment (seanjedi) [reply pandas/tests/frame/methods/test_reset_index.py]: Thanks for the feedback, this should be updated as of: 471d3b36bdd79c05b2f85bba4a30af9aab899549

File: pandas/tests/indexes/multi/test_analytics.py [hunk 4]
```diff
@@ -16,7 +16,7 @@ def test_shift(idx):
 
     # GH8083 test the base class for shift
     msg = "This method is only implemented for DatetimeIndex, PeriodIndex and "
-    "TimedeltaIndex; Got type MultiIndex"
+    # TimedeltaIndex; Got type MultiIndex
```
[the remark this answers is not in the document]
Comment (seanjedi) [reply pandas/tests/indexes/multi/test_analytics.py]: Thanks for the feedback, this should be updated as of: 471d3b36bdd79c05b2f85bba4a30af9aab899549

File: pandas/tests/indexing/interval/test_interval_new.py [hunk 5]
```diff
@@ -162,8 +162,10 @@ def test_loc_with_overlap(self, indexer_sl):
         result = indexer_sl(ser)[Interval(1, 5) : Interval(3, 7)]
         tm.assert_series_equal(expected, result)
 
-        msg = "'can only get slices from an IntervalIndex if bounds are"
-        " non-overlapping and all monotonic increasing or decreasing'"
+        msg = (
+            "'can only get slices from an IntervalIndex if bounds are"
+            + " non-overlapping and all monotonic increasing or decreasing'"
```
[the remark this answers is not in the document]
Comment (seanjedi) [reply pandas/tests/indexing/interval/test_interval_new.py]: Ahh I see, alright I will do that then!  
This should be updated as of: 471d3b36bdd79c05b2f85bba4a30af9aab899549

File: pandas/tests/io/parser/common/test_read_errors.py [hunk 6]
```diff
@@ -241,8 +241,10 @@ def test_bad_header_uniform_error(all_parsers):
     data = "+++123456789...\ncol1,col2,col3,col4\n1,2,3,4\n"
     msg = "Expected 2 fields in line 2, saw 4"
     if parser.engine == "c":
-        msg = "Could not construct index. Requested to use 1 "
-        "number of columns, but 3 left to parse."
+        msg = (
+            "Could not construct index. Requested to use 1 "
+            + "number of columns, but 3 left to parse."
+        )
```
[the remark this answers is not in the document]
Comment (seanjedi) [reply pandas/tests/io/parser/common/test_read_errors.py]: Ahh I see, alright I will do that then!  
This should be updated as of: 471d3b36bdd79c05b2f85bba4a30af9aab899549

**Review by MarcoGorelli — APPROVED** [review]
Thanks @seanjedi !

**Pull request merged**